.github/workflows/release.yml

```diff
@@ -1,80 +1,81 @@
-name: Release Buildpack
+name: Release Buildpacks
 
 on:
   workflow_dispatch:
 
 permissions:
   contents: write
-
-env:
-  BUILDPACK_DOCKER_REPO: docker.io/heroku/buildpack-python
-  CARGO_TERM_COLOR: always
+  pull-requests: write
 
 jobs:
-  # Releases the buildpack to Docker Hub and registers it with the CNB Buildpack Registry.
-  # This release process intentionally does not create a .cnb file release for now, since
-  # there are currently no use-cases that need it for Python.
-  release:
-    name: Release heroku/python
-    runs-on: pub-hk-ubuntu-22.04-large
-    steps:
-      # Setup
-      - name: Checkout
-        uses: actions/checkout@v3
-      - name: Install musl-tools
-        run: sudo apt-get install musl-tools --no-install-recommends
-      - name: Update Rust toolchain
-        run: rustup update
-      - name: Install Rust linux-musl target
-        run: rustup target add x86_64-unknown-linux-musl
-      - name: Rust Cache
-        uses: Swatinem/rust-cache@v2.5.0
-      - name: Install libcnb-cargo
-        run: cargo install libcnb-cargo
-      - name: Install Pack CLI
-        uses: buildpacks/github-actions/setup-pack@v5.2.0
-      - name: Install yj and crane
-        uses: buildpacks/github-actions/setup-tools@v5.2.0
-      - name: Login to Docker Hub
-        uses: docker/login-action@v2
-        with:
-          registry: docker.io
-          username: ${{ secrets.DOCKER_HUB_USER }}
-          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+  detect:
```

Comment:
each workflow is parameterized to work with a single buildpack if you give it the correct metadata so, technically, you could drop this job but then you'd have to replace it with another that did roughly the same thing so i doubt it would be worth it.

also, for consistency with other buildpacks i think it makes sense that each project maintains the same workflow.